src/healthcareapis/HISTORY.rst

```diff
@@ -3,6 +3,24 @@
 Release History
 ===============
 
+0.4.0
+++++++
+
+* Bump version to 2021-11-01 stable.
+* `az healthcareapis service` Add new argument `--login-servers` to support adding login servers to the service instance.
+* `az healthcareapis service` Add new argument `--oci-artifacts` to support specifying open container initiative artifacts.
+* `az healthcareapis private-endpoint-connection` Deprecate argument `--private-link-service-connection-state-actions-required` since sdk version 2020-03-30
```

Comment:
Remove “since sdk version 2020-03-30" as "sdk" is an implementation concept and CLI customers don't know that.